### DISCUSSION TITLE
App is not starting in browser after flet build web

### DISCUSSION THREAD

**Discussion by alegro96** [flet-dev/flet#2894] [Q&A]
### Question

I builded my App with "flet build web" and published it local with "python -m http.server --directory build/web". 
I tried to start it in the browser but there is only a grey window and not my App. The browsers dev-tool gives the attached error message.

![image](https://github.com/flet-dev/flet/assets/135384121/dd0178e0-644a-4df3-8c42-1ad13f609e2c)

Can anyone help me with this? Is the not installed 'request'-module the problem? How can I solve this?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Flet makes use of Pyodide for static websites, and [it doesn't](requests) yet support the use of the requests package.
Instead of using requests you can make use of the [Pyodide.http](https://pyodide.org/en/stable/usage/api/python-api/http.html). Example usage [here](https://dev.to/openhacking/3-ways-to-implement-network-requests-in-pyodide-4e5n).
